src/browser/transport.js

```diff
@@ -57,7 +58,8 @@ Transport.prototype.post = function(accessToken, options, payload, callback, req
   var writeData = stringifyResult.value;
   var method = 'POST';
   var url = _.formatUrl(options);
-  _makeZoneRequest(accessToken, url, method, writeData, callback, requestFactory);
+  var transportOptions = _getMakeZoneRequestOptions(options);
```

Comment:
There are a couple other ways to handle this that might be simpler.

It would be OK to just pass the `options` as is, without filtering to just include the timeout. Or it would also be OK to pass the timeout as an argument. There's only one option in this case, so it doesn't necessarily need to be wrapped in an options argument.

I would favor either of those approaches over the extra code to support `_getMakeZoneRequestOptions`. We're always looking for opportunities to keep the browser package smaller, and rebuilding the options object here isn't strictly necessary.

Reply:
Ok, removed _getMakeZoneRequestOptions and passed only timeout